idemp-rest-test/testing/src/api/receiveProducts.ts

```diff
@@ -0,0 +1,8 @@
+import { ReceiveProduct } from '../types/org.bandahealth.idempiere.rest';
+import { DocumentApi } from './documents';
+
+class ReceiveProductApi extends DocumentApi<ReceiveProduct> {
+	entityName = 'receiveproducts';
+}
+
+export const receiveProductApi = new ReceiveProductApi();
```

Comment:
```suggestion
export const receiveProductApi = new ReceiveProductApi();

```

Reply:
Both lines are the same. what am I to change?

Reply:
Add a new line at the end of the file.